server/src/test/java/org/opensearch/extensions/action/ExtensionActionRequestTests.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions.action;
+
+import org.opensearch.common.bytes.BytesReference;
+import org.opensearch.common.io.stream.BytesStreamInput;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.nio.charset.StandardCharsets;
+
+public class ExtensionActionRequestTests extends OpenSearchTestCase {
+
+    public void testExtensionActionRequest() throws Exception {
+        String action = "test-action";
```

Comment:
(personal preference) I like using variable names starting with "expected".

Reply:
Sure, I like it. Updated!